code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/SchemaType.java

```diff
@@ -0,0 +1,86 @@
+/*
+  Copyright 2023 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+ */
+
+package com.adobe.marketing.mobile.messaging;
+
+public enum SchemaType {
+    UNKNOWN(0),
+    HTML_CONTENT(1),
+    JSON_CONTENT(2),
+    RULESET(3),
+    INAPP(4),
+    FEED(5),
+    NATIVE_ALERT(6),
+    DEFAULT(7);
```

Comment:
let's call this `DEFAULT_CONTENT` since it has a specific context for personalization responses

Reply:
Done